### ISSUE TITLE
Currency in product price changes when changing language.

### ISSUE THREAD

**Issue by afernandes** [simplcommerce/SimplCommerce#410]
**Currency in product price changes when changing language.**
Currently: Language: EN = $ 1,800 when changing to pt-BR = R$ 1,800.
Correct: Do not change currency.

The best would be if there was an option to select the currency in the product register.
Or make the nopCommerce style price change.
use the website http://www.ecb.europa.eu/stats/eurofxref/eurofxref-daily.xml
to obtain exchange rates

**Comment by afernandes** [comment]
Hey! this question is very important!
The price of products change as the site language changes.
In the product register you should have the option to select the currency.

**Comment by afernandes** [comment]
I want to start creating the currency module.
okay for you?

**Comment by thiennn** [comment]
I see most of the time, ecommerce websites are targeted and deployed to each country, so on the site there are 1 language and 1 currency. 

But yes, you are welcomed. It will be best if we can build an module for it and plug it in on demand

**Comment by GoranHalvarsson** [comment]
@afernandes Hello :) Did you start with the currency module?

**Comment by skfd** [comment]
@thiennn Canadian shops need English and French, but only CAD currency. Lots of other examples.

**Comment by afernandes** [comment]
Unfortunately, I was out of time, but I believe that Monday I will be able to start the creation of the module.
I'm thinking of creating something similar to what exists in nopCommerce, but feel free to start the creation of the module or if you have suggestions and ideas for the module.

**Comment by afernandes** [comment]
work in progress

**Comment by amohsyn** [comment]
@afernandes how long will you take to finish the module??

**Comment by thiennn** [comment]
I am considering adding a simple helper for formatting the price. Let say in the admin, there will be a setting for currency:
- The culture you want to display for currency
- The routing type ex. how many decimal places
The helper will format the price base on those settings. There should be 1 currency for entire the site regarding to the language

**Comment by afernandes** [comment]
I already did some things.  but I don't know if I made the best decisions.  I relied heavily on nopcommerce.

I did the Exchange part but the helper is missing

https://github.com/afernandes/SimplCommerce/tree/ExchangeRateProvider?files=1



I later discovered this project, which seems interesting to me.  but I haven't had time to test yet

https://github.com/zpbappi/money

To make something simple that works for everything, it would be interesting to register the coin in the product itself, near the price.  then put default currency in vendor.
